common/src/main/java/org/opensearch/ml/common/MLModelGroup.java

```diff
@@ -52,14 +53,15 @@ public class MLModelGroup implements ToXContentObject {
 
     private Instant createdTime;
     private Instant lastUpdatedTime;
+    private String tenantId;
 
 
     @Builder(toBuilder = true)
     public MLModelGroup(String name, String description, int latestVersion,
                         List<String> backendRoles, User owner, String access,
                         String modelGroupId,
                         Instant createdTime,
-                        Instant lastUpdatedTime) {
+                        Instant lastUpdatedTime, String tenantId) {
```

Comment:
Nit: spotless should have wrapped this on a new line?

Reply:
not really I think. I applied `./gradlew spotlessApply`, but that didn't change anything. You can see old code `List<String> backendRoles, User owner, String access,` in the same way.

Reply:
Huh. Weird. Spotless hasn't been doing things on this repo that I expect (I find trailing whitespace often).

Seems weird that it would put modelGroupId and createdTime on separate lines but then combine the next two arguments on one line. I think Spotless is broken :-|